pyxona/core.py

```diff
@@ -598,6 +598,26 @@ def _read_tracking2(self):
         # stack all coordinates into an array
         all_pos = np.concatenate([np.stack(xy, axis=-1) for xy in pos_filtered], axis=1)
 
+        # Remove NaN values at the endpoints
+        for c in range(0, all_pos.shape[1]):
```

Comment:
just being picky but prefer to omit the first number if 0 - that is use `range(all_pos.shape[1])` instead